backend/classify.functions.php

```diff
@@ -130,3 +130,30 @@ function ipHash($hostname, $debug = false) {
     }
 }
 
+function getCurrentCountry($hostname, $debug = false) {
+
+    try {
+        if ( $debug ) {
+            echo $hostname."\n";
+        }
+        if (!class_exists('GeoIp2\Database\Reader')) {
+            throw new \Exception('Class GeoIp2\Database\Reader not found');
+        }
+        $reader = new Reader(__DIR__.'/GeoLite2-City.mmdb');
+        $record = $reader->city($hostname);
+        if ( $debug > 1 ) {
+            var_dump($record);
+        }
+        $country   = $record->country->isoCode;
+
+    } catch (\Exception $e) {
+        echo '<!-- '.$e->getMessage().' -->'."\n";
+        $country   = false;
+    }
+
+    if ( $debug ) {
+        echo 'Country: "'.$country.'"'."\n";
```

Comment:
@btkostner That file has always done that, and only with the debug mode enabled, it's what you see at https://elementary.io/backend/classify.get.php

There's no sensitive data there, only the individuals detected country/region.